src/IssueViz.Security/Hotspots/HotspotsList/HotspotsControl.xaml

```diff
@@ -87,14 +87,15 @@
                         </DataTemplate>
                     </DataGridTemplateColumn.CellTemplate>
                 </DataGridTemplateColumn>
-                <DataGridTemplateColumn CanUserSort="True" SortMemberPath="CategoryDisplayName" Header="Category" Width="Auto"  MinWidth="74">
+                <!-- Category is not currently populated in the view model. See #4565. -->
+                <!--<DataGridTemplateColumn CanUserSort="True" SortMemberPath="CategoryDisplayName" Header="Category" Width="Auto"  MinWidth="74">
```

Comment:
Just removed the column from the XAML. I added a ticket to correctly populate it at some point in the future.